duckduckgo/macos-browser
# Update privacy dashboard with child entity logos

Task/Issue URL: https://app.asana.com/0/0/1202836875374614/f
Tech Design URL:
CC:

**Description:**
Adds icons for Google and Facebook child entities (added in the next blocklist update).

Note I'm omitting URLbar logos in this change, as 1) this is a short term change until the new dashboard design is implemented, and 2) we don't have dark-mode designs for these logos.

**Steps to test this PR:**

1. Switch to the 'beta' blocklist (change 'v2.1' to 'beta' [here](https://github.com/more-duckduckgo-org/macos-browser/blob/develop/DuckDuckGo/Configuration/ConfigurationDownloading.swift#L44)).
2. Visit test sites. Verify that for the new entities the correct icon is shown:
  - "Google Ads (Google)": youtube.com, android.com
  - "Google Analytics (Google)": blogger.com
  - "Youtube (Google)": https://privacy-test-pages.glitch.me/privacy-protections/youtube-click-to-load/ (in not blocked section).
  - "Instagram (Facebook)": https://www.cinemablend.com/movies/see-the-four-hobbits-from-lotr-reuniting-at-a-restaurant-and-totally-making-the-day-of-a-nearby-fan 

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
